libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -1,4 +1,3 @@
-{-# LANGUAGE DeriveAnyClass #-}
```

Comment:
I remember that we don't want this extension, but I don't remember why.  I might be wrong.

But the json instances created weren't very nice:

```
*Wire.API.Team.Feature Data.Aeson> encode TeamFeatureTTLUnlimited
"{\"tag\":\"TeamFeatureTTLUnlimited\"}"
*Wire.API.Team.Feature Data.Aeson> encode (TeamFeatureTTLSeconds 3)
"{\"contents\":3,\"tag\":\"TeamFeatureTTLSeconds\"}"
```

Which is a nice case for adding golden tests for new types as well as keeping the old ones, come to think of it.  :)